src/reducers/__tests__/reducers-test.js

```diff
@@ -0,0 +1,20 @@
+import 'react-native'
+import reducers, { initialState } from '../index.js'
+import { setUser, setState } from '../../actions/index'
+
+it('sets user', () => {
+  let fakeUser = { avatar: 'Some avatar', display_name: 'Me'}
+  expect(reducers(initialState, setUser(fakeUser))).toMatchSnapshot()
+})
+
+it('unsets user', () => {
+  expect(reducers(initialState, setUser({}))).toMatchSnapshot()
+})
+
+it('sets state from initial state', () => {
+  expect(reducers(initialState, setState('isConnected', true))).toMatchSnapshot()
+})
+
+it('sets a new state', () => {
```

Comment:
These match with the snapshots directly? Ex: this matches with what is expected in the object found at line 1 of `reducers-test.js.snap`?